Cabal/Distribution/C2Hs.hs

```diff
@@ -0,0 +1,56 @@
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE FlexibleContexts #-}
+
+-- Based on https://github.com/gtk2hs/gtk2hs/blob/master/tools/src/Gtk2HsSetup.hs#L414
```

Comment:
That's GPLd repository, we cannot "base" anything on that code.

Reply:
Oh, good point.